doc/source/conf.py

```diff
@@ -205,6 +207,8 @@
     'preamble': r''' 
         \DeclareUnicodeCharacter{2588}{=}
         \DeclareUnicodeCharacter{258E}{|} 
+        \DeclareUnicodeCharacter{274C}{$\times$} 
+        \DeclareUnicodeCharacter{2705}{$\checkmark$} 
```

Comment:
What are these symbols that latex can't parse so they have to be replaced?

Reply:
The cross and tick parks in the "Saving and Loading" page:

![image](https://user-images.githubusercontent.com/32061685/169835559-94dbc7ba-3379-4192-8f81-a4c2b5b4bd01.png)